frontend/apps/app/app/api/webhook/github/route.ts

```diff
@@ -1,22 +1,20 @@
 import { verifyWebhookSignature } from '@/libs/github/api'
 import { supportedEvents, validateConfig } from '@/libs/github/config'
 import { handleInstallation } from '@/libs/github/webhooks/installation'
-import { handlePullRequest } from '@/libs/github/webhooks/pull-request'
+import { savePullRequestTask } from '@/src/trigger/jobs'
 import type { GitHubWebhookPayload } from '@/types/github'
 import { type NextRequest, NextResponse } from 'next/server'
 
 export const POST = async (request: NextRequest): Promise<NextResponse> => {
   try {
-    const payload = await request.text()
+    const payload = await request.json()
+    const signature = request.headers.get('x-hub-signature-256') ?? ''
 
-    const signature = request.headers.get('x-hub-signature-256') || ''
-    if (!verifyWebhookSignature(payload, signature)) {
-      return NextResponse.json({ error: 'Invalid signature' }, { status: 401 })
-    }
-
-    const data = JSON.parse(payload) as GitHubWebhookPayload
+    validateConfig()
```

Comment:
This line is unnecessary, because Because it is done at server startup.

ref: https://github.com/liam-hq/liam/blob/ignore-twice-build/frontend/apps/app/instrumentation.ts#L13

Reply:
oh, good!

Reply:
I'll fix that this week 🙏